addons/packages/vsphere-csi/2.6.2/bundle/config/overlays/add-namespace.yaml

```diff
@@ -0,0 +1,7 @@
+#@ load("/values.star", "values")
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: #@ values.vsphereCSI.namespace
```

Comment:
will it block pkg installation in the second time or upgrade pkg when namespace is already there ? e.g. install pkg --> install namespace --> remove package --> install pkg again

Reply:
I have tested the given scenario and it is working fine during upgrade pkg